gtk/DetailsDialog.cc

```diff
@@ -1936,22 +1983,26 @@ void appendScrapeInfo(tr_tracker_view const& tracker, time_t const now, Gtk::Tex
 
         if (tracker.lastScrapeSucceeded)
         {
-            gstr << gtr_sprintf(
-                _("Tracker had %s%'d seeders and %'d leechers%s %s ago"),
-                success_markup_begin,
-                tracker.seederCount,
-                tracker.leecherCount,
-                success_markup_end,
-                timebuf);
+            gstr << fmt::format(
+                // {markup_begin} and {markup_end} should surround the seeder/leecher text
+                _("Tracker had {markup_begin}{seeder_count} {seeders_ngettexted} and {leecher_count} {leechers_ngettexted}{markup_end} {time_span} ago"),
```

Comment:
This is kind of awkward. Maybe there should be a dual ngettext here for the two plurals.

Reply:
I went with 118447dc59f660ac42ea9ae3234b42fc7c12c140 and it does cover the cases but I'm not proud of it & welcome better ideas

Reply:
This is not going to work well for languages with more than 2 plural forms, previous approach was actually better in this regard :) Also, two bound placeholder values are now unused.

Reply:
Re-revised in ac521d62e0b9f13ab12c1ee8e21f3e5dc7692525